docs/site/Decorators_repository.md

```diff
@@ -133,19 +130,134 @@ Register a general relation.
 _This feature has not yet been released in alpha form. Documentation will be
 added here as this feature progresses._
 
-#### Specific Relation Decorator
+#### BelongsTo Decorator
 
 Syntax:
 
-- `@belongsTo`
-- `@hasOne`
-- `@hasMany`
+- `@belongsTo(targetResolver: EntityResolver<T>, definition?: Partial<BelongsToDefinition>)`
```

Comment:
Let's leave out the bullet points here and also merge the description into two sentences:
```
Syntax: `@belongsTo(targetResolver: EntityResolver<T>, definition?: Partial<BelongsToDefinition>)`

Many-to-one or one-to-one connection between models e.g. a `Todo` model belongs to a `TodoList` model. See [BelongsTo relation](BelongsTo-relation.md) for more details.
```

Reply:
+1 to leave out the bullet point here.